lib/bgsite_official/home.ex

```diff
@@ -24,6 +24,17 @@ defmodule BgsiteOfficial.Home do
     end
   end
 
+  def bump_likes(%Websites{} = website, likes) do
+    website = website.id
+    query = from(w in Websites, where: w.websites_id == ^website)
```

Comment:
Please, change it with the generic `get_website!`, this is the cause for our next errors related to `website.id`. Here is the proper form: 
https://github.com/N-Patarov/bgsite_official/blob/add-likes/lib/bgsite_official/home.ex#L91